source/Government.h

```diff
@@ -155,8 +157,10 @@ class Government {
 	double reputationMax = std::numeric_limits<double>::max();
 	double reputationMin = std::numeric_limits<double>::lowest();
 	std::map<int, double> penaltyFor;
-	std::map<const Outfit*, int> illegals;
-	std::map<const Outfit*, bool> atrocities;
+	std::map<const Outfit*, int> illegalOutfits;
+	std::map<const std::string, int> illegalShips;
+	std::map<const Outfit*, bool> atrocityOutfits;
+	std::map<const std::string, bool> atrocityShips;
```

Comment:
Looks like this is what's causing the Mac issue.
```suggestion
	std::map<std::string, int> illegalShips;
	std::map<const Outfit*, bool> atrocityOutfits;
	std::map<std::string, bool> atrocityShips;
```

Reply:
but why??

Reply:
The elements of the map (`std::pair<const std::string, bool>`) are not copy-assignable.
That `std::pair` cannot be used in an expression like: `pair1 = pair2` because it cannot do: `first = other.second` because `first` is a `const std::string`.

Reply:
then why only Mac gives an error?

Reply:
Different standard library implementations.

Reply:
Isn't the standard library supposed to be _standard_?